macros/Compat.ejs

```diff
@@ -99,23 +123,57 @@ a string to appear in the table cell, like "Yes", "No" or "?"
 function getVersionString(versionInfo) {
   switch (versionInfo) {
     case null:
-      return `<span title="${localize(compatStrings, 'supportsShort_unknown_title')}"
-              style="color: rgb(255, 153, 0);">${localize(compatStrings, 'supportsShort_unknown')}</span>`;
-      break;
+      return `<span title="${localize(compatStrings, 'supportsShort_unknown_title')}">
+      ${localize(compatStrings, 'supportsShort_unknown')}</span>`;
+    break;
     case true:
-      return `<span title="${localize(compatStrings, 'supportsShort_yes_title')}"
-              style="color: #888">${localize(compatStrings, 'supportsShort_yes')}</span>`;
-      break;
+      return `<span title="${localize(compatStrings, 'supportsShort_yes_title')}">
+      ${localize(compatStrings, 'supportsShort_yes')}</span>`;
+    break;
     case false:
-      return `<span title="${localize(compatStrings, 'supportsLong_no')}"
-              style="color: #f00">${localize(compatStrings, 'supportsShort_no')}</span>`;
-      break;
+      return `<span title="${localize(compatStrings, 'supportsLong_no')}">
+      ${localize(compatStrings, 'supportsShort_no')}</span>`;
+    break;
     default:
       return versionInfo;
   }
 }
 
 /*
+Given the support information for a browser, this returns
+a CSS class to apply to the table cell.
+
+`supportData` is a (or an array of) support_statement(s)
+*/
+function getSupportClass(supportInfo) {
+  let cssClass = 'unknown-support';
+
+  if (Array.isArray(supportInfo)) {
+    // the first entry should be the most relevant/recent and will be treated as "the truth"
```

Comment:
I think this should be the first. When I look at the table cells, I think I like seeing the most relevant compat data first. So, for example, if there is unpreffed support from Firefox 51, I think that is the most relevant information. Then only after that I'll see that there was also preffed support from versions 34-50.